cvat/apps/engine/views.py

```diff
@@ -2003,6 +2005,16 @@ def self(self, request):
         serializer = serializer_class(request.user, context={ "request": request })
         return Response(serializer.data)
 
+    @action(detail=True, methods=['GET', 'PATCH'])
```

Comment:
First of all, it is the wrong location for the method. I will recommend moving the code into limit_manager. Also the 'limits' endpoind is better than 'limitations' from my perspective.

Reply:
/api/limits?user_id=id
/api/limits?org_id=id
/api/limits?project_id=id

Reply:
I agree with both points.

Reply:
Actually, I don't see big reasons in this specific endpoint now, because it looks too narrow for our current usecases. Probably, there are no cases where we would like to get or edit 1 single limitation. Getting or editing just a single limitation is ineffective, because it would require a separate request for each operation. My suggestion is to consider 1 user or 1 org limits as just a single structure. A user or an org is not going to have an arbitrary number of limitations, which populated from somewhere else (unlike projects or users, for example).

Reply:
Actually, I don't see big reasons in this specific endpoint now, because it looks too narrow for our current usecases. Probably, there are no cases where we would like to get or edit 1 single limitation. Getting or editing just a single limitation is ineffective, because it would require a separate request for each operation. My suggestion is to consider 1 user or 1 org limits as just a single structure. A user or an org are not going to have an arbitrary number of limitations, which populated from somewhere else (unlike projects or users, for example). An example of such could be an option to control org limits on a user basis, which is not going to be supported in the near future.

Reply:
One thing is that all of these "list" requests will return at most 1 structure. Thus, it can be better to add:

`/users/limits` (GET, PATCH)
`/orgs/limits` (GET, PATCH)

or

`/limits/user/{id}` (GET, PATCH)
`/limits/org/{id}` (GET, PATCH)